src/RecordsModel.php

```diff
@@ -82,4 +82,14 @@ public function addRecord(
 
         $query->execute();
     }
+
+    // soft delete record by ID
+    public function removeRecord(int $id)
+    {
+        $query = $this->db->prepare("UPDATE `records` SET `deleted` = 1 WHERE `id` = :idNum LIMIT 1");
+
+        $query->bindParam('idNum', $id);
+
+        $query->execute();
```

Comment:
You should return execute, the boolean it provides will tell you whether or not the query worked